omroneipApp/src/utilityFunctions.cpp

```diff
@@ -0,0 +1,1101 @@
+#include "drvOmroneip.h"
+    
+omronUtilities::omronUtilities(drvOmronEIP *ptrDriver)
+{
+    pDriver = ptrDriver;
+    pasynUserSelf = pDriver->pasynUserSelf;
+    driverName = pDriver->driverName;
+}
+
+drvInfoMap omronUtilities::drvInfoParser(const char *drvInfo)
```

Comment:
That is one long function! Please consider refactoring this into some smaller and more digestible (and testable!) chunks...

Reply:
For example in the for-loop, each of the `if (i == X){}` sections could probably be a function. There are 4 of them, I think...